autoload/ale/lsp.vim

```diff
@@ -412,7 +412,7 @@ function! s:SendInitMessage(conn) abort
     \               'completion': {
     \                   'dynamicRegistration': v:false,
     \                   'completionItem': {
-    \                       'snippetSupport': v:false,
+    \                       'snippetSupport': v:true,
```

Comment:
Unfortunatelly I think this needs more evaluation.  I did some tests and when this is enabled ALEImport breaks for me.
If enablinb snippetSupport is not critical for vscodejson I would recommend reverting this to get vscodejson merged and then handle adding snippetSupport on a separate issue.

Reply:
Absolutely not critical. Let me fix it.

Reply:
@hsanson BTW, how ALEImport breaks for you? What filetype you were workin on? I want to reproduce the problem and see if there is a fix.

Reply:
Simple Java HelloWorld.java managed via Gradle:

```
public class HelloWorld {
    public static void main(String[] argv) {
        LocalDate today = LocalDate.now();
        System.out.println("Hello gradle world at " + today);
    }
}
```

1. Put the cursos in LocalDate class and invoke :ALEImport.
2. ALE should add the import statement "import java.time.LocalDate" at the beginning of the file.
3. With the snippetSupport enabled this does not work.